front/src/components/BreadcrumbNavigation.tsx

```diff
@@ -35,6 +37,10 @@ export function BreadcrumbNavigation({ items }: IBreadcrumbNavigationProps) {
   const { globalPreferences } = useContext(GlobalPreferencesContext);
   const orgName = globalPreferences.organisation?.name;
   const baseName = globalPreferences.selectedBase?.name;
+  const { isLoading: isGlobalStateLoading } = useLoadAndSetGlobalPreferences();
+
+  if (isGlobalStateLoading) return <APILoadingIndicator />;
```

Comment:
could you replace this with a simple skeleton. It looks weird when the global loading indicator is shown instead of just one line in the view. Maybe ther is already a fitting skeleton in /components/Skeletons.tsx